wordpress/wp-content/plugins/memberful-wp/src/acl.php

```diff
@@ -129,7 +129,21 @@ function memberful_wp_user_feeds($user_id) {
  * @return array member's subscriptions
  */
 function memberful_wp_user_plans_subscribed_to( $user_id ) {
-  return get_user_meta( $user_id, 'memberful_subscription', TRUE );
+  return memberful_wp_get_user_meta_for_acl( $user_id, 'memberful_subscription' );
+}
+
+/**
+ * `get_user_meta` may return `false` or `""` if the given $user_id
+ * doesn't exist or is invalid. Because the expected ACL interface is an array,
+ * we return an empty one.
+ */
+function memberful_wp_get_user_meta_for_acl($user_id, $meta_key, $single = TRUE) {
+  $meta = get_user_meta($user_id, $meta_key, $single);
+
+  if ($meta == false || $meta == "")
```

Comment:
According to [PHP's type comparison tables](https://www.php.net/manual/en/types.comparisons.php), `"" == false` is `true` so we can use:

```suggestion
  if ($meta == false)
```

Reply:
Good to know, thanks!